delivery/models/project.py

```diff
@@ -154,6 +155,17 @@ def __del__(self):
                 " Check token expiry date (`dds auth info`).")
             raise
 
+    def get_dds_version(self):
+        """
+        Get the DDS version been used
+        """
+
+        result = subprocess.run(['dds', '--version'], stdout=subprocess.PIPE)
+        version_stdout = result.stdout.strip().decode()     # e.g version_stdout = Data Delivery System, version 2.6.1
+        version = (version_stdout.split("version")[-1])
```

Comment:
```suggestion
        version = (version_stdout.split("version")[-1]).strip()
```
This might contain a trailing space at the beginning (e.g. ` 2.6.1`)